app/views/components/_dashboard.html.erb

```diff
@@ -0,0 +1,18 @@
+<div class="dashboard">
+  <div class="dashboard-grocery">
+    <h3>You are missing only:</h3>
+    <ul class="grocery-list">
+      <% grocery_list.each do |item| %>
+        <li class="grocery-list-item"><%= item.name.capitalize %></li>
```

Comment:
item.name.titleize works better as it capitalizes both words. name.capitalize only capitalizes the first word